web/eslint.config.js

```diff
@@ -25,5 +25,13 @@ export default tseslint.config(
       ...reactHooks.configs.recommended.rules,
       "react-refresh/only-export-components": ["warn", { allowConstantExport: true }],
     },
+  },
+  {
+    files: ["cypress.config.ts"],
+    rules: {
+      "@typescript-eslint/no-require-imports": "off",
+      "@typescript-eslint/no-unsafe-assignment": "off",
+      "@typescript-eslint/no-unsafe-call": "off",
+    },
```

Comment:
I got some frustrating typescript and eslint conflicting errors, so I had to add this to allow the cypress config to pass. This should be ok as it is just the standard cypress config that doesn't contain any critical app logic.